packages/app/src/feature-flags/features/index.ts

```diff
@@ -102,6 +102,11 @@ export const features: EnvironmentConfig = {
     [Environment.Production]: false,
     [Environment.AlphaProduction]: true,
   },
+  [Feature.MessagesDraftStatus]: {
+    [Environment.Development]: true,
+    [Environment.Production]: false,
+    [Environment.AlphaProduction]: true,
```

Comment:
I noticed some issues. After merging this PR the issue still will be occurring. Maybe we should change the feature flag to false? 🤔 `AlphaProduction` is commitment to the 1.5.0/1.6.0, no 2.0.0 (next major). 

However, it will be hard to manage test features until new pure isn't released.

```suggestion
    [Environment.AlphaProduction]: false,
```

Reply:
AS discussed, we should leave the flag as it is